src/self-model.scm

```diff
@@ -315,6 +315,50 @@
 					(ListLink (VariableNode "$face-id"))))
 		)))
 
+;; Return true if someone recognizable but not yet acknowledged face appears.
+(DefineLink
+	(DefinedPredicateNode "Did someone recognizable arrive?")
+	(SatisfactionLink
+		(VariableList
+			(TypedVariableLink
+				(VariableNode "$face-id")
+				(TypeNode "ConceptNode"))
+			(TypedVariableLink
+				(VariableNode "$recog-id")
+				(TypeNode "ConceptNode"))
+			(TypedVariableLink
+				(VariableNode "$face-id-nn")
+				(TypeNode "NumberNode")))
+		(AndLink
+			; If someone is visible...
+			(PresentLink (EvaluationLink (PredicateNode "visible face")
+					(ListLink (VariableNode "$face-id-nn"))))
+			; but not yet acknowledged...
+			(AbsentLink (EvaluationLink (PredicateNode "acked face")
+					(ListLink (VariableNode "$face-id-nn"))))
```

Comment:
I'm pretty sure you want to delete the above two lines. A face gets "acked" when she looks at the face.  Looking at them is like a non-verbal "hello".  Instead, you probably want to "ack" and say  hello at the same time. 

In addition, the "ack" design should be reviewed, and maybe fixed. Right now, "ack" is simply the turn-to-look-at animation; but really, it should be that, or maybe a chin-push animation, or a look-and-nod animation (the opposite of a chin-push), or maybe a look+smile animation, etc. Likewise, the words can be "hello", "what's up", "que pasa", "yo"...